code/game/machinery/GBPvendor.dm

```diff
@@ -0,0 +1,125 @@
+
+/obj/machinery/GBP_vendor
+	name = "engineering point redemption"
+	desc = "Who's a good boy?"
+	icon = 'icons/obj/vending.dmi'
+	icon_state = "liberationstation"
+	density = 1
+	anchored = 1
+	var/GBP = 0
+	var/GBPspent = 0
+	var/GBPearned = 0
+	var/list/prize_list = list( //if you add something to this, please, for the love of god, use tabs and not spaces.
+		new /datum/GBP_equipment("Whiskey",				/obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey,		50,	1),
+		new /datum/GBP_equipment("Cigar",				/obj/item/clothing/mask/cigarette/cigar/havana,						50,	1),
+		new /datum/GBP_equipment("Soap",				/obj/item/weapon/soap/nanotrasen,									100,	1),
+		new /datum/GBP_equipment("Insulated Gloves",				/obj/item/clothing/gloves/color/yellow,					100,	1),
+		new /datum/GBP_equipment("Fulton Beacon",		/obj/item/fulton_core,												50,	1),
+		new /datum/GBP_equipment("Fulton Pack",			/obj/item/weapon/extraction_pack,									200,	1),
+		new /datum/GBP_equipment("Space Cash",			/obj/item/stack/spacecash/c1000,									250,	1),
+		new /datum/GBP_equipment("50 metal sheets",			/obj/item/stack/sheet/metal/fifty,								500,	1),
+		new /datum/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/glass/fifty,								500,	1),
+		new /datum/GBP_equipment("50 cardboard sheets",			/obj/item/stack/sheet/cardboard/fifty,						500,	1),
+		new /datum/GBP_equipment("Hardsuit x3",			/obj/item/clothing/suit/space/hardsuit,								750,	3),

```

Comment:
Holy shit, almost everything below this needs to be 1x, or not be there, why are there so many op items..?
Don't even think of telling me that this is balanced by power consumption because while I havn't played the new SM engine yet I know that the old days allowed for EASY 2 million watts with no effort, and that's noslips under 4 minutes.